### PR TITLE
fix: Rename performPermissionRequest

### LINKED ISSUE #13413: Rename performPermissionRequest
It should indicates that it returns whether the permission was performed or not. Intuitively, I'd guess it indicates whether we have permission or not, or something similar.

_Originally posted by @david-allison in https://github.com/ankidroid/Anki-Android/pull/13261#discussion_r1111391658_

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Purpose / Description
Rename `performPermissionRequest` .

## Fixes
Fixes #13413 

## Approach
Rename the method name.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
Seems almost....too easy

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt [hunk 1]
```diff
@@ -97,7 +97,7 @@ class MultimediaEditFieldActivity : AnkiActivity(), OnRequestPermissionsResultCa
         finishWithoutAnimation()
     }
 
-    private fun performPermissionRequest(field: IField): Boolean {
+    private fun onRequestMultimediaPermissionsResult(field: IField): Boolean {
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt:100]: I don't think this is the name to use as `onWordWord...` type of method names are usually associated with callbacks, which this method isn't. I think a better name would be `hasPerformedPermissionRequestForField`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt (hunk 1 above)
Comment (pratyaksh1610) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/activity/MultimediaEditFieldActivity.kt]: Done

**Review by lukstbit — APPROVED** [review]
Great, thanks!

**Pull request merged**